hacspec/src/array.rs

```diff
@@ -288,7 +288,7 @@ macro_rules! generic_array {
             }
 
             #[cfg_attr(feature="use_attributes", primitive(hacspec))]
-            pub fn len(&self) -> usize {
+            pub fn length() -> usize {
```

Comment:
Yeah no problem, there is another version of `len(&self)` elsewhere anyway to implement `SeqTrait`. However this function is now `library` instead of primitive, since it just returns a constant integer :)